tests/unit/plugins/test_colcon.py

```diff
@@ -951,6 +970,7 @@ class BuildTestCase(ColconPluginTestBase):
 
     package_scenarios = [
         ("one package", {"colcon_packages": ["my_package"]}),
+        ("two packages", {"colcon_packages": ["my_package", "other_package"]}),
```

Comment:
Is there a reason you added this scenario?

Reply:
I was trying to test if the `package-ignore` actually works by having multiple packages at first. But it is not really necessary I guess. should I remove it?